common/src/main/java/gov/cms/ab2d/common/service/JobServiceImpl.java

```diff
@@ -54,6 +56,7 @@ public void cancelJob(String jobUuid) {
         Job job = getJobByJobUuid(jobUuid);
```

Comment:
You can add the job's UUID to `MDC` prior to this line

Reply:
Wouldn't this already be in place by the line in BulkDataAccessAPI at the beginning of deleteRequest ?

Reply:
You're right; disregard.